### PR TITLE
Update clickhouse app

### PR DESCRIPTION
**Pull request by kvaps**
as part of https://github.com/aenix-io/cozystack/issues/73

* Add users management
* Remove logs volume

### CODE REVIEW

File: packages/apps/clickhouse/templates/clickhouse.yaml [hunk 1]
```diff
@@ -3,16 +3,27 @@ kind: "ClickHouseInstallation"
...
   name: "{{ .Release.Name }}"
 spec:
+  {{- with .Values.size }}
   defaults:
     templates:
       dataVolumeClaimTemplate: data-volume-template
-      logVolumeClaimTemplate: log-volume-template
+  {{- end }}
   configuration:
+    {{- with .Values.users }}
```
Comment (gecube) [inline packages/apps/clickhouse/templates/clickhouse.yaml:12]: what will happen if we are removing user from `values.yaml`?

**Review by gecube — CHANGES_REQUESTED** [review]

File: packages/apps/clickhouse/templates/clickhouse.yaml (hunk 1 above)
Comment (kvaps) [reply packages/apps/clickhouse/templates/clickhouse.yaml:12]: The whole block will be omited

Comment (gecube) [reply packages/apps/clickhouse/templates/clickhouse.yaml:12]: I don't mean on Helm level, but what will happen on application level.

Comment (kvaps) [reply packages/apps/clickhouse/templates/clickhouse.yaml:12]: The cluster will be created without configured users.

All these user values just translated into `users.xml` file

Comment (gecube) [reply packages/apps/clickhouse/templates/clickhouse.yaml:12]: I am not talking about INITIALIZE phase, but rather - about lifecycle. So when the cluster already created and you are changing the values.

Comment (kvaps) [reply packages/apps/clickhouse/templates/clickhouse.yaml:12]: ClickHouse will recognize the file automatically without restarting and you can login using a command like the following:

https://altinity.com/blog/goodbye-xml-hello-sql-clickhouse-user-management-goes-pro

**Review by gecube — APPROVED** [review]

**Pull request merged**